force-app/main/default/objects/HOT_Request__c/fields/ServiceResourceWithAgreement1__c.field-meta.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>ServiceResourceWithAgreement1__c</fullName>
+    <deleteConstraint>SetNull</deleteConstraint>
+    <externalId>false</externalId>
+    <label>Service Resource With Agreement 1</label>
+    <referenceTo>ServiceResource</referenceTo>
```

Comment:
Kanskje legge på et filter om at det er en aktiv tjenesteressurs. Samme som vi la til på Assigned Resource.

Reply:
`You reached the maximum number of allowed active filtered lookups (5) on Request (26:13)`